### ISSUE TITLE
Autoreconf error for rofi-calc

### ISSUE THREAD

**Issue by sarmong** [pacstall/pacstall#1127]
### Describe the bug

I have a pacscript for [rofi-calc](https://github.com/svenstaro/rofi-calc). It was working fine before 5.0, but now that I migrated it to new version, it no longer works.

This is the error I get after running `autoreconf`

```
/usr/bin/libtoolize: 1: eval: hookable_fns+=: not found
libtoolize:   error: 'func_options_prep' does not accept hook functions.
autoreconf: error: libtoolize failed with exit status: 1
```

### To reproduce

1. Download [this pacscript](https://github.com/sarmong/pacstall-sarmong/blob/5f3b807936fc0094efa405334dc9b4ab0ef21246/packages/rofi-calc/rofi-calc.pacscript)
2. `pacstall -I rofi-calc.pacscript`

### Expected behavior

No errors

### Screenshots

-

### OS version

Debian 12

### Pacstall version

5.1.1

**Comment by Elsie19** [comment]
Can you try putting the line:
```bash
cd "${_archive}"
```
above line https://github.com/sarmong/pacstall-sarmong/blob/5f3b807936fc0094efa405334dc9b4ab0ef21246/packages/rofi-calc/rofi-calc.pacscript#L17

**Comment by sarmong** [comment]
Yeah, I forgot to put it in when pushing, but it fails on prepare step anyways.

**Comment by Elsie19** [comment]
![image](https://github.com/pacstall/pacstall/assets/58742515/8a2a2b1f-3941-4b0f-a440-0c6bba8d138d)
Works for me. What issue do you get?

**Comment by sarmong** [comment]
It runs fine for me, I get errors only in pacstall


https://github.com/pacstall/pacstall/assets/42828375/72003a32-7c97-4e66-9586-15f7dd54e30a

**Comment by Elsie19** [comment]
Run `pacstall -V`.

**Comment by sarmong** [comment]
I already mentioned pacstall version in the original post. It is 5.1.1.

**Comment by Elsie19** [comment]
Can you copy the pacscript and run `docker run -it --net=host --privileged ghcr.io/pacstall/pacstall:latest bash`, and test it in there?

**Comment by sarmong** [comment]
it works. Did it succeed for you on Ubuntu? Could it be Debian-specific issue? Is there a Debian-based docker image to test with?

**Comment by Elsie19** [comment]
> it works. Did it succeed for you on Ubuntu? Could it be Debian-specific issue? Is there a Debian-based docker image to test with?

Yes, you can do any of the following:

```bash
$ docker run -it --net=host --privileged ghcr.io/pacstall/debian-stable:develop-978e2905 bash
$ docker run -it --net=host --privileged ghcr.io/pacstall/debian-unstable:develop-978e2905 bash
$ docker run -it --net=host --privileged ghcr.io/pacstall/debian-testing:develop-978e2905 bash
```

**Comment by sarmong** [comment]
Yep, same error on debian-stable docker as I have.

**Comment by Elsie19** [comment]
Can you try installing `libtool-bin` and `libtool` in the docker before running the script?

**Comment by sarmong** [comment]
Didn't help.

**Comment by sarmong** [comment]
Hey, any updates?

**Comment by Elsie19** [comment]
No, we couldn't find anything besides simply disallowing debian, unless that works for you.
